cumulusci/core/github.py

```diff
@@ -444,6 +453,12 @@ def format_github3_exception(exc: Union[ResponseError, TransportError]) -> str:
         sso_error_msg = check_github_sso_auth(exc)
         user_warning = scope_error_msg + sso_error_msg
 
+    if isinstance(exc, ConnectionError):
+        if "self signed certificate" in str(exc.exception):
+            user_warning = SELF_SIGNED_WARNING
+        else:
+            user_warning = exc.msg
```

Comment:
❓ : Would it make sense to return `None` in the case where we have a `ConnectionError` that isn't a self signed certificate? In that case we don't have any advice to offer the user as it may not be related to the GitHub API (maybe they're on a spotty mobile network, etc).